src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -304,6 +308,24 @@ internal static ArgumentOutOfRangeException InvalidCommandType(CommandType value
             return InvalidEnumerationValue(typeof(CommandType), (int)value);
         }
 
+        internal static ArgumentOutOfRangeException InvalidCommandBehavior(CommandBehavior value)
+        {
+#if DEBUG
+            if ((0 <= (int)value) && ((int)value <= 0x3F))
+            {
+                Debug.Assert(false, "valid CommandType " + value.ToString());
+            }
+#endif
```

Comment:
Nit: could just be a Debug.Assert with the inverted condition rather than having Assert(false, ...) and using #if DEBUG.

I'm also a little confused by the condition being checked, as it differs from the condition being checked in ValidateCommandBehavior below; I'd have expected them to be identical.